cypress/integration/smokeTest.js

```diff
@@ -25,75 +33,172 @@ describe('Smoke test', () => {
   });
 
   it('formats', () => {
-    cy.visit('http://localhost:9000/').reload();
-
     typeCode('<Foo><Foo><Bar/>');
     assertCodePaneLineCount(1);
     formatCode();
     assertCodePaneLineCount(6);
   });
 
-  it('url handling - code (base64 encoding)', () => {
-    cy.visit(
-      'http://localhost:9000/#?code=PEZvbz48Rm9vPjxCYXIvPjwvRm9vPjwvRm9vPg'
-    ).reload();
-
-    assertFirstFrameContains('Foo');
-    assertCodePaneContains('<Foo><Foo><Bar/></Foo></Foo>');
+  it('frames are interactive', () => {
+    getFirstFrame().click('center');
```

Comment:
Isn't this function is async? Also this test has no assertions. Is that intentional?

Reply:
Just moved the test in the PR. The very act of clicking on the frame is the test. If something is blocking the frame this test fails.